src/plugins/data/common/interpreter/registries.ts

```diff
@@ -17,13 +17,20 @@
  * under the License.
  */
 
-// All types must be universal and be castable on the client or on the server
+/* eslint-disable max-classes-per-file */
 import { get } from 'lodash';
-import { getType } from '@kbn/interpreter/common';
+import { Registry } from '@kbn/interpreter/src/common';
```

Comment:
if we plan to put all registries in all file, then we should probably put Renderers here as well, but i think we can keep them separate, i don't see any benefit from joining them.

Reply:
In the latest iteration I've put all `@kbn/interpreter` code data plugin depends on in "public" into `src/plugins/data/expressions/interpreter.ts`. Effectively, NP plugin currently does not depend on `@kbn/interpreter` anymore. Plan is to refactor that file quite a bit in the future.